client/src/actions/team.js

```diff
@@ -10,11 +10,11 @@ import {
   CREATE_TEAM_MEMBER,
   UPDATE_TEAM_MEMBER,
   DELETE_TEAM_MEMBER,
-  ADD_TEAM,
   GET_GH_ACCESS_TOKEN,
   DEPLOY_SUCCESS,
   DEPLOY_FAIL,
   UPDATE_TEAM,
+  REGISTRATE_SUCCESS,
```

Comment:
Might want to change REGISTRATE_SUCCESS as "register" naming convention is used in other files?